ci/travis/install-bazel.sh

```diff
@@ -121,3 +121,18 @@ build --remote_upload_local_results=false
 EOF
   fi
 fi
+
+if [ "${TRAVIS-}" = true ]; then
+  # Inject the helper function to bashrc, so in future invocation
+  # we get a shorthand to export logs to a temp file on each
+  # test/build event.
+  cat <<'EOF' >> ~/.bashrc
+bazel_export_log_opt() {
+  mkdir -p /tmp/bazel_event_logs
+  echo "--build_event_json_file $(mktemp /tmp/bazel_event_logs/bazel_log.XXXXX)"
```

Comment:
Just echo this into `~/.bazelrc` instead (like I did with `--google_credentials` on line 115)? You really don't want to call this everywhere Bazel is called; `--config=ci` should be sufficient.

Reply:
The issue here is that we want each test run to write to a unique file otherwise they will override each other. I can't add this kind of dynamic generation in bazelrc. If we only add:

```
test:ci --build_event_json_file="/tmp/log"
```

then the each different `bazel test`s will overwrite each other instead of appending to the same file.

Reply:
Oh I see, thanks, I misunderstood. What I would do in this case is to redefine `bazel` instead. That way you can isolate the change to one location instead of changing the entire code. Here's an example that might work:
```
bazel() {
  case "${1-}" in
    --*) echo "Bazel startup options not yet implemented for this thunk" 1>&2; false;;
    test) command bazel "$1" --build_event_json_file "$(mktemp /tmp/bazel_event_logs/bazel_log.XXXXX)" "${@:2}";;
    *) command bazel "$@";
  esac
}
```

Reply:
Ah that's smart. I'll try that!